### ISSUE TITLE
Can't add more keywords/tags after issue has been funded with a bounty

### ISSUE THREAD

**Issue by knocte** [gitcoinco/web#4203]
**Describe the bug**
Can't add more keywords/tags after issue has been funded with a bounty

**To Reproduce**
1. Create a github issue.
2. Create a bounty associated to the issue above, with some keywords/tags.
3. After ETH has been confirmed in the blockchain (bounty successfully published), navigate to the issue on gitcoin website, as the funder, and click to edit the details of the issue.

**Expected behavior**
Keywords/tags field should be editable, in order to be able to add more.

**Additional context**
Using Brave+Metamask.

**Comment by Skyge** [comment]
Is there a necessary to show issue detail the same time?

**Comment by knocte** [comment]
I don't understand your question, what do you mean?

**Comment by Skyge** [comment]
Emmm, you know, when we fill in the `Fund Issue` then it will show keywords/tags, so do you want to edit keywords/tags alone?

**Comment by knocte** [comment]
> do you want to edit keywords/tags alone?

"alone"? Not 100% sure what you mean here so I'll explain what happened to me, maybe you understand this way: I funded this issue recently (https://gitcoin.co/issue/psantosl/p2pcopy/4/2817), and after doing it, I realised I forgot to include the tag "C#" (only added "linux" and "webrtc"), but it seems that the UI doesn't let me change it now. I want to add that keyword to make it attractive to C# developers.
